fre/pp/ppyaml_test/am5-yamls.yaml

```diff
@@ -0,0 +1,12 @@
+## FRE properties
+projectID: "am5"
+FRE_VERSION: &fre_version "canopy"
+AM5_VERSION: &am5_version "2022.01" #"am5f7b12r1"
+stem: "am5/2022.01" # stem=projectID/version
```

Comment:
What do you think about organizing the properties under a header instead, like:

```
properties:
  projectID: "am5"
  FRE_VERSION: &fre_version "canopy"
  AM5_VERSION: &am5_version "2022.01"
  stem: "am5/2022.01"
```

In those 4 properties, projectID and stem have just one value, and FRE_VERSION and AM5_VERSION have another definitions, `&fre_version` and `&am5_versions`. Why the difference, and what is the `&` for? (Pardon my yaml ignorance..)